tests/test_decompositions.py

```diff
@@ -210,6 +227,35 @@ def test_gaussian_transform(self):
         state = self.eng.run()
         self.assertAllAlmostEqual(state.cov(), self.S@self.S.T*self.hbar/2, delta=self.tol)
 
+    def test_graph_embed(self):
+        """Test that embedding a traceless adjacency matrix A
+        reults in the property Amat/A = c J, where c is a real constant,
+        and J is the all ones matrix"""
+        self.logTestName()
+        self.eng.reset()
+        q = self.eng.register
+
+        with self.eng:
+            GraphEmbed(self.A) | q
+
+        state = self.eng.run()
+        Amat = self.eng.backend.circuit.Amat()
+
+        ratio = np.real_if_close(Amat[self.num_subsystems:, self.num_subsystems:]/self.A)
```

Comment:
Question: why do we index only the bottom-right block?

Reply:
The matrix Amat is a 2Nx2N block diagonal matrix, by construction composed of [[B^, 0], [0, B]]. So it is sufficient to only consider the bottom right NxN block.

Reply:
then it shouldn't hurt to add another some other asserts for the other 3 blocks to make sure that the `by construction` steps worked as intended